src/services/dealer-price/proto/services/price/v1/price_service_pb.d.ts

```diff
@@ -0,0 +1,245 @@
+// package: services.price.v1
+// file: services/price/v1/price_service.proto
+
+/* tslint:disable */
+/* eslint-disable */
+
+import * as jspb from "google-protobuf"
+
+export class GetExchangeRateForImmediateUsdBuyRequest extends jspb.Message {
+  getAmountInSatoshis(): number
```

Comment:
what is the goal of this function that don't take any input argument? versus the function below?

Reply:
if this is because the argument is optional, we probably want to make it mandatory (not an expert in grpc but I guess we could do that)

Reply:
These are 4 generated code files in that folder from the one proto file I defined. It exist to statically strongly type the grpcs. Will also validate versioning if ever it evolves.

Reply:
yeah looked at it and it seems there is no way in GRPC to make an argument mandatory (unlike Graphql) so this is why those 2 methods are being generated